docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/graph-service-administration/graph-metrics.md

```diff
@@ -0,0 +1,82 @@
+# Graph Metrics
+
+## 介绍
+
+目前，**Nebula Graph** 支持通过 HTTP 方式来获取 Graph Service 层的基本性能指标。
+
+每一个性能指标都由三部分组成，分别为指标名、统计类型、时间范围。
+
+| counter\_name | statistic\_type | time_range |
+| ----  |  ----|-------|
+
+### 指标名
+
+每个指标名都由接口名加指标名构成，目前支持获取如下接口：
+
+```text
+graph_storageClient: 通过 storageClient 发送的请求，需要同时向多个 storage 并发多条消息时，按一次统计
+graph_metaClient: 通过 metaClient 发送的请求
+graph_graph_all: 客户端向 graph 发送的请求，当一条请求包含多条语句时，按一条计算
+graph_insertVertex: 插入点
+graph_insertEdge: 插入边
+graph_deleteVertex: 删除点
+graph_deleteEdge: 删除边  //代码还没加
+graph_updateVertex: 更新点的属性
+graph_updateEdge: 更新边的属性
+graph_go: 执行 go 命令
+graph_findPath: 查找最小路径或者全路径
+graph_fetchVertex: 获取点属性，不统计获取点的总数，只统计执行命令的数量
+graph_fetchEdge: 获取边属性，不统计边的总数，只统计执行命令的数量
+```
+
+每一个接口都有三个性能指标，分别为延迟(单位为 us)、QPS、发生错误的 QPS，后缀名如下：
+
+```text
+_latency
+_qps
+_error_qps
+```
+
+将接口名和相应指标连接在一起即可获得完整的指标名，例如 `graph_insertVertex_latency`、`graph_insertVertex_qps`、`graph_insertVertex_error_qps`、分别代表插入一个点的延迟、QPS 和发生错误的 QPS。
+
+### 统计类型
+
+目前支持的统计类型有 SUM、COUNT、AVG、RATE 和 P 分位数 (P99，P999， ... ，P999999)。其中：
+
+- `_latency` 和 `_error_qps` 这两类后缀的指标，支持 SUM、COUNT、AVG、RATE，但不支持 P 分位；
```

Comment:
`_latency` support P99 etc, but `_error_qps` and `_qps` does not. You'd better check all doc